ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -30,7 +35,9 @@
 
     <p class="is-label">Which secrets would you like us to sync?</p>
     <p class="sub-text">
-      Select a KV engine mount and path to sync a secret to the destination.
+      Select a KV engine mount and path to sync a secret to the
+      {{@destination.typeDisplayName}}
```

Comment:
Should this be `@destination.name` to match the header?

Reply:
This is the description so this seemed like a good place to include the destination type rather than repeat the destination name. Otherwise the icon is the only type indicator.